server/github.go

```diff
@@ -87,13 +87,13 @@ func (s *Server) GetPullRequestFromGithub(ctx context.Context, pullRequest *gith
 	return pr, nil
 }
 
-func (s *Server) GetIssueFromGithub(ctx context.Context, repoOwner, repoName string, ghIssue *github.Issue) (*model.Issue, error) {
+func (s *Server) GetIssueFromGithub(ctx context.Context, ghIssue *github.Issue) (*model.Issue, error) {
 	issue := &model.Issue{
-		RepoOwner: repoOwner,
-		RepoName:  repoName,
-		Number:    *ghIssue.Number,
-		Username:  *ghIssue.User.Login,
-		State:     *ghIssue.State,
+		RepoOwner: ghIssue.GetRepository().GetOwner().GetLogin(),
+		RepoName:  ghIssue.GetRepository().GetName(),
+		Number:    ghIssue.GetNumber(),
+		Username:  ghIssue.GetUser().GetLogin(),
+		State:     ghIssue.GetState(),
```

Comment:
This change should be carefully reviewed, I expect these were redundant. That's why I removed them. However, it's not so clear in the [docs](https://docs.github.com/en/developers/webhooks-and-events/github-event-types).